core/network/types/block_attributes.hpp

```diff
@@ -32,60 +32,121 @@ namespace kagome::network {
   };
 
   /**
-   * Block attributes as set of bits
+   * Block attributes
    */
-  struct BlockAttributes {
-    enum { kBitCount = 8 };
-    std::bitset<kBitCount> attributes;
+  class BlockAttributes {
```

Comment:
I have a strong feeling that some generalised version of this should exist in boost.

Reply:
Could you explain that, please?